index.html

```diff
@@ -0,0 +1,182 @@
+    <link rel="icon" href="src/img/tram.png"/>
+    <title>Travel</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-light bg-dark pe-5 media">
+            <div class="container-fluid d-flex justify-content-end">
+                <div>
+                    <a class="navbar-brand logo" href="#"><img class="facebook" src="src/img/facebook.svg" alt=""></a>
+                    <a class="navbar-brand logo" href="#"><img class="twitter" src="src/img/twitter.svg" alt=""></a>
+                </div>
+
+                <div class="collapse navbar-collapse" id="navbarSupportContent">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0 justify-content-end">
+                        <li class="nav-item">
+                            <a class="nav-link text-white" href="#">English</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link text-white" href="#">My page</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link text-white" href="#">Logout</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+
+        <nav class="navbar navbar-expand-lg navbar-light container">
+            <div class="container-fluid">
+                <a class="navbar-brand logo" href="index.html"><i class="fas fa-tram fa-3x">travel</i></a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+                    aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0 justify-content-end">
+                        <li class="nav-item">
+                            <a class="nav-link" aria-current="page" href="about.html">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Program</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Join</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Sponsor</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">News</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="btn btn-outline-danger border-4">CC Campaign</a>
+                        </li>
+                    </ul>
+                </div>
```

Comment:
The mobile menu should have a close (X) button that closes the menu. Please add the close (X) button with its functionality.